### PR TITLE
PIM-5951: Extensibility issue with viewElement in content tab

### PR DESCRIPTION
**Pull request by fitn**
This PR adds the possibility to plug new jobs on the JobNameVisibilityChecker. It also removes the EE jobs from the CE.

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Y
| Added Behats                      | N
| Changelog updated                 | N
| Review and 2 GTM                  | Waiting for :D 
| Micro Demo to the PO (Story only) |
| Migration script                  | N
| Tech Doc                          | N

### CODE REVIEW

File: src/Pim/Bundle/ImportExportBundle/PimImportExportBundle.php [hunk 1]
```diff
@@ -24,6 +25,9 @@ public function build(ContainerBuilder $container)
     {
         $container
             ->addCompilerPass(new RegisterJobTemplatePass())
-            ->addCompilerPass(new RegisterJobParametersFormsOptionsPass(new ReferenceFactory()));
+            ->addCompilerPass(new RegisterJobParametersFormsOptionsPass(new ReferenceFactory()))
+            ->addCompilerPass(new RegisterJobNameVisibilityCheckerPass(
+                ['pim_connector.job_name.csv_product_export', 'pim_connector.job_name.xlsx_product_export']
```
Comment (juliensnz) [inline src/Pim/Bundle/ImportExportBundle/PimImportExportBundle.php]: could you use DI parameters ?

File: src/Pim/Bundle/ImportExportBundle/Resources/config/view_elements/job_profile.yml [hunk 2]
```diff
@@ -57,7 +57,7 @@ services:
            - 'pim_import_export.job_profile.tab.job_content'
            - '%pim_import_export.view_element.job_profile.tab.job_content.template%'
        calls:
-           - [ addVisibilityChecker, ['@pim_import_export.view_element.visibility_checker.job_name', {job_names: ['csv_product_export', 'csv_published_product_export', 'xlsx_product_export', 'xlsx_published_product_export']}] ]
+           - [ addVisibilityChecker, ['@pim_import_export.view_element.visibility_checker.job_name'] ]
```
Comment (nidup) [inline src/Pim/Bundle/ImportExportBundle/Resources/config/view_elements/job_profile.yml]: :+1: :fire:

**Comment by juliensnz** [comment]
GTM

**Comment by damien-carcel** [comment]
GTM

File: src/Pim/Bundle/ImportExportBundle/PimImportExportBundle.php [hunk 3]
```diff
@@ -24,6 +25,9 @@ public function build(ContainerBuilder $container)
     {
         $container
             ->addCompilerPass(new RegisterJobTemplatePass())
-            ->addCompilerPass(new RegisterJobParametersFormsOptionsPass(new ReferenceFactory()));
+            ->addCompilerPass(new RegisterJobParametersFormsOptionsPass(new ReferenceFactory()))
+            ->addCompilerPass(new RegisterJobNameVisibilityCheckerPass(
```
Comment (SamirBoulil) [inline src/Pim/Bundle/ImportExportBundle/PimImportExportBundle.php]: 👍  didn't think of it :)

**Pull request merged**